src/components/Wallets.test.jsx

```diff
@@ -9,6 +9,18 @@ import Wallets from './Wallets'
 
 jest.mock('../libs/JmWalletApi')
 
+jest.mock('react-i18next', () => ({
```

Comment:
Instead of mocking `react-i18next` you can use the `I18nextProvider` as a wrapper:

https://github.com/cstenglein/raspiblitz-web/blob/master/src/components/Shared/SendModal/SendOnChain/SendOnChain.test.tsx#L3

Reply:
Cool, that's a very good point. Updated. ✅